### ISSUE TITLE
envelopesApi.update: Error: INVALID_REQUEST_BODY

### ISSUE THREAD

**Issue by by12380** [docusign/docusign-node-client#136]
In an attempt to resend a document using the node api client, the code below throws error status 400 - INVALID REQUEST BODY.

```
await envelopesApi.update(dsJwtAuth.accountId, envelopeId, {
    resendEnvelope: true
})
```

However, doing the following works:

```
await envelopesApi.update(dsJwtAuth.accountId, envelopeId, {
    resendEnvelope: true,
    envelope: {}
})
```

Fix suggestions:
1. Update node-client-api documentation for `update(accountId, envelopeId, opts, callback)` to say opts.envelope is required

OR

2. Update api/EnvelopesApi.js, line 3861 to be `var postBody = opts['envelope'] || {} ;`

**Comment by LarryKlugerDS** [comment]
Thank you for the bug report. I have filed internal report DCM-3369.

**Comment by acooper4960** [comment]
Hi @by12380 This issue has been resolved starting in version 4.10.1 and 5.8.1

**Comment by shierro** [comment]
Can confirm that this is working, thanks! might be nice to have this as an example on examples project
